sirepo/package_data/static/js/sirepo-plotting.js

```diff
@@ -4088,21 +4099,32 @@ SIREPO.app.directive('parameterPlot', function(appState, focusPointService, layo
                 });
                 updateYLabel();
                 plots.forEach(function(plot, i) {
-                    if (cachedPlotVisibilty(i, $scope.modelName)) {
+                    if (cachedPlotVisibility(i, $scope.modelName)) {
                         setPlotVisible(i, ! isPlotVisible(i));
                     }
                 });
 
-                $scope.$on(
-                    $scope.modelName + '.changed',
-                    () => {
+                function handlePlotSelection() {
```

Comment:
I'm not sure whether it's worth making a function here since it is defined and then immediately invoked, and only in that one place...